packages/lorcana-engine/src/types/game-state.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * Game State Types
+ *
+ * Core game state types for Lorcana engine.
+ */
+
+/** Player identifier */
+export type PlayerId = string;
+
+/** Card instance identifier */
+export type CardId = string;
+
+/** Zone identifier */
+export type ZoneId = "deck" | "hand" | "play" | "inkwell" | "discard";
```

Comment:
Let's also create one zone for phased out cards, as cards that are shifted or action card while playing